ilastik/__init__.py

```diff
@@ -25,31 +25,31 @@
 ################################
 # # Add Submodules to sys.path ##
 ################################
-import os
 import h5py
-import time
-from typing import Optional, Iterable, List
+from typing import Optional, List
 from pkg_resources import parse_version
```

Comment:
We probably shouldn't depend on modules from `setuptools` in runtime, such as `pkg_resources`. It could be replaced by the [`packaging`](https://packaging.pypa.io/en/latest/version.html) package.